lib/db.js

```
@@ -721,6 +766,16 @@ module.exports = function (
     return this.pool.post('/account/' + account.uid.toString('hex') + '/verifyEmail')
   }
 
+  DB.prototype.verifyTokens = function (tokenVerificationId, tokenData) {
+    log.trace({ op: 'DB.verifyToken', tokenVerificationId: tokenVerificationId })
+    return this.pool.post('/tokens/' + tokenVerificationId.toString('hex') + '/verify',
+        unbuffer(
+          tokenData,
+          'inplace'
+        )
+      )
+  }
```

Do we need to do something here to make it throw `invalidToken` error if nothing matched the submitted id?